### ISSUE TITLE
link attribute in image field type

### ISSUE THREAD

**Issue by szepeviktor** [michaeluno/admin-page-framework#244]
`link` could take into consideration the "Link To" drop-down in Media selection: None/Media URL/Page URL/Custom

Now it always stores the page URL.

**Comment by michaeluno** [comment]
Maybe `link_to` or something can be added to the `attributes_to_store` argument for it. I'll see how it can be done.

**Comment by michaeluno** [comment]
As discussed in https://github.com/michaeluno/admin-page-framework/issues/239#issuecomment-188837464, it does not seem to be so simple to achieve.

**Comment by szepeviktor** [comment]
It is very common that you need a bunch of images with links.

**Comment by szepeviktor** [comment]
Maybe a new base type? `image_link`?

**Comment by michaeluno** [comment]
I agree with doing it with a new field type. You should try creating a [custom field type](http://admin-page-framework.michaeluno.jp/en/v3/package-AdminPageFramework.FieldType.html).
